backend/typescript/rest/schedulingRoutes.ts

```diff
@@ -101,6 +101,31 @@ schedulingRouter.get("/volunteers/:volunteerId?", async (req, res) => {
   }
 });
 
+schedulingRouter.get("/pickup/:isPickUp", async (req, res) => {
```

Comment:
@jessiiepeng are all the other endpoints still working? I know it had to be in a certain order for `/scheduling/volunteers` route to work, so just making sure adding this did not break anything else.

Reply:
yep! tested and it still works with other endpoints